src/SubscriberWorker/Program.cs

```diff
@@ -0,0 +1,59 @@
+using System.Text.Json.Serialization;
+using Blumchen.Configuration;
+using Blumchen.Serialization;
+using Blumchen.Subscriptions;
+using Blumchen.Workers;
+using Commons;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Polly.Retry;
+using Polly;
+using SubscriberWorker;
+
+
+#pragma warning disable CS8601 // Possible null reference assignment.
+Console.Title = typeof(Program).Assembly.GetName().Name;
+#pragma warning restore CS8601 // Possible null reference assignment.
+
+
+
+AppDomain.CurrentDomain.UnhandledException += (_, e) => Console.Out.WriteLine(e.ExceptionObject.ToString());
+TaskScheduler.UnobservedTaskException += (_, e) => Console.Out.WriteLine(e.Exception.ToString());
+
+var cancellationTokenSource = new CancellationTokenSource();
+var builder = Host.CreateApplicationBuilder(args);
+
+builder.Services
+    .AddBlumchen<SubscriberWorker<UserCreatedContract>, UserCreatedContract>()
```

Comment:
**CONCERN:** Does that mean that we need to call AddBlumchen for Contracts separately? My preference would be to have a single AddBlumchen with additional lambda as param for configuring dependencies. See example how we did it in Marten: https://martendb.io/configuration/hostbuilder.html#bootstrapping-marten

Reply:
Every Worker<T> is bound to specific message type via its messgehandler( which can take care of different message types). 
This way we are providing different service workers by message type(s) - a useful strategy to deal with different publication technologies/retry strategies/external dependencies.
We might review it later with a further iteration